content/en/synthetics/api_tests/ssl_tests.md

```diff
@@ -19,12 +19,19 @@ further_reading:
 
 ## Overview
 
-SSL tests allow you to **proactively monitor the validity and expiration of your SSL/TLS certificates** to ensure secure connections between your key services and your users. If your certificate is about to expire or becomes compromised, Datadog sends you an **alert with details on the failure**, allowing you to quickly pinpoint the root cause of the issue and fix it.
```

Comment:
Why deleting the bold?
I'm not against it, just want to understand your thoughts

Reply:
While our Contributing Style Guide (https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md) states that bold subjectively points readers to something important, I use bold specifically to call out UI elements. In this Overview section, the bold is fine for action verbs, but I'd rather not dilute the purpose of bold.